core/controllers/base.py

```diff
@@ -110,20 +110,36 @@ class NotLoggedInException(Exception):
     class InvalidInputException(Exception):
         """Error class for invalid input on the user side (error code 400)."""
 
+        pass
+
+
     class UnauthorizedUserException(Exception):
         """Error class for unauthorized access."""
 
+        pass
+
+
     class PageNotFoundException(Exception):
         """Error class for a page not found error (error code 404)."""
 
+        pass
+
+
+
```

Comment:
@Hudda, extra newline? Please do a self-review before assigning reviewers. Also, are we expecting pre-push lint checks to fail on this line?

Reply:
Okay, I will keep that in mind. I'll self-review first from next time. I think we don't have a check for this.